docs/interfaces.md

```diff
@@ -176,6 +176,12 @@ playback.
 
 * Auto-Connect: yes
 
+### pluggable-storage
+
+Can access filesystems mounted on USB drives or SD cards
```

Comment:
Perhaps this is better-- `Can access files from pluggable storage in /media and /run/media`